src/server/AccountsServer.js

```diff
@@ -4,9 +4,21 @@ import { defaultServerConfig } from '../common/defaultConfigs';
 import { AccountsError } from '../common/errors';
 import AccountsCommon from '../common/AccountsCommon';
 import type { AccountsOptionsType } from '../common/AccountsCommon';
+import type { DBDriverInterface } from './DBDriver';
+import type UserObjectType from '../common/UserObjectType';
+
+
+export type UserCreationInputType = {
```

Comment:
Maybe move that into the common folder so both could the client and the server will use the same type?

Also a possibility will be to create declaration files.